deploy/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: nginx-gateway
+description: NGINX Kubernetes Gateway
+type: application
+version: 0.1.0
+appVersion: "0.4.0"
```

Comment:
this look inconsistent. `version: 0.1.0` but `appVersion: "0.4.0"` with "

Reply:
It's from the best practices - version should be semver, appVersion should be a string:

> Note that the appVersion field is not related to the version field. It is a way of specifying the version of the application. For example, the drupal chart may have an appVersion: "8.2.1", indicating that the version of Drupal included in the chart (by default) is 8.2.1. This field is informational, and has no impact on chart version calculations. Wrapping the version in quotes is highly recommended. It forces the YAML parser to treat the version number as a string. Leaving it unquoted can lead to parsing issues in some cases. For example, YAML interprets 1.0 as a floating point value, and a git commit SHA like 1234e10 as scientific notation.